File: Web.Admin/src/api/services/department/models/QueryDepartmentDetailOutput.ts
/**
 * Fast.Admin.Service.Department.Dto.QueryDepartmentDetailOutput 获取部门详情输出
 */
export interface QueryDepartmentDetailOutput {
  /**
   * 部门Id
   */
  departmentId?: number;
  /**
   * 机构Id
   */
  orgId?: number;
  /**
   * 机构名称
   */
  orgName?: string;
  /**
   * 父级Id
   */
  parentId?: number;
  /**
   * 父级名称
   */
  parentName?: string;
  /**
   * 父级Id集合
   */
  parentIds?: Array<number>;
  /**
   * 父级名称集合
   */
  parentNames?: Array<string>;
  /**
   * 部门名称
   */
  departmentName?: string;
  /**
   * 部门编码
   */
  departmentCode?: string;
  /**
   * 联系人
   */
  contacts?: string;
  /**
   * 电话
   */
  phone?: string;
  /**
   * 邮箱
   */
  email?: string;
  /**
   * 排序
   */
  sort?: number;
  /**
   * 备注
   */
  remark?: string;
  /**
   * 创建者用户名称
   */
  createdUserName?: string;
  /**
   * 创建时间
   */
  createdTime?: Date;
  /**
   * 更新者用户名称
   */
  updatedUserName?: string;
  /**
   * 更新时间
   */
  updatedTime?: Date;
  /**
   * 更新版本控制字段
   */
  rowVersion?: number;
}

File: Api.Server/src/Admin.Service/Department/Dto/QueryDepartmentDetailOutput.cs
// Copyright © 2018-Present 小方
// SPDX-License-Identifier: Apache-2.0
// 
// 本文件依据 Apache License 2.0 授权，完整条款见仓库根目录 LICENSE。
// 本软件按“原样”提供，相关免责声明及责任限制以许可证及适用法律为准。
// 版权来源、合法使用与二次开发责任说明见仓库根目录 README.md。

namespace Fast.Admin.Service.Department.Dto;

/// <summary>
/// 获取部门详情输出
/// </summary>
public class QueryDepartmentDetailOutput
{
    /// <summary>
    /// 部门Id
    /// </summary>
    public long DepartmentId { get; set; }

    /// <summary>
    /// 机构Id
    /// </summary>
    public long OrgId { get; set; }

    /// <summary>
    /// 机构名称
    /// </summary>
    public string OrgName { get; set; }

    /// <summary>
    /// 父级Id
    /// </summary>
    public long ParentId { get; set; }

    /// <summary>
    /// 父级名称
    /// </summary>
    public string ParentName { get; set; }

    /// <summary>
    /// 父级Id集合
    /// </summary>
    [SugarColumn(IsJson = true)]
    public List<long> ParentIds { get; set; }

    /// <summary>
    /// 父级名称集合
    /// </summary>
    [SugarColumn(IsJson = true)]
    public List<string> ParentNames { get; set; }

    /// <summary>
    /// 部门名称
    /// </summary>
    public string DepartmentName { get; set; }

    /// <summary>
    /// 部门编码
    /// </summary>
    public string DepartmentCode { get; set; }

    /// <summary>
    /// 联系人
    /// </summary>
    public string Contacts { get; set; }

    /// <summary>
    /// 电话
    /// </summary>
    public string Phone { get; set; }

    /// <summary>
    /// 邮箱
    /// </summary>
    public string Email { get; set; }

    /// <summary>
    /// 排序
    /// </summary>
    public int Sort { get; set; }

    /// <summary>
    /// 数据公开
    /// </summary>
    public bool DataPublic { get; set; }

    /// <summary>
    /// 备注
    /// </summary>
    public string Remark { get; set; }

    /// <summary>
    /// 创建者用户名称
    /// </summary>
    public string CreatedUserName { get; set; }

    /// <summary>
    /// 创建时间
    /// </summary>
    public DateTime? CreatedTime { get; set; }

    /// <summary>
    /// 更新者用户名称
    /// </summary>
    public string UpdatedUserName { get; set; }

    /// <summary>
    /// 更新时间
    /// </summary>
    public DateTime? UpdatedTime { get; set; }

    /// <summary>
    /// 更新版本控制字段
    /// </summary>
    public long RowVersion { get; set; }
}
